### PR TITLE
changelog

### PR DESCRIPTION
**Pull request by jburel**
Add CHANGELOG and README

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- Include Bio-Formats 6.1.0-m1.
+- Add omero.pixeldata.memoizer.dir to set BioFormatsCache.
+- Note current deprecations in model mapping files.
+- Add @Deprecation annotations to ome.model.* classes.
+- Replace http by https when suitable.
+
+5.5.0-m7 (April 2019)
+---------------------
+
+- Reduce the number of declaration for Bio-Formats dependencies.
```
Comment (mtbc) [inline CHANGELOG.md]: declarations

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- Convert Unicode in properties file to non-literal.
+- Reduce the number of declaration for Bio-Formats dependencies.
```
Comment (mtbc) [inline CHANGELOG.md]: duplicate of above

File: README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Like any Java project, include the jar into your classpath.
+
+## Build from source
+
+The compilation, testing, launch, and delivery of the application are
+automated by means of a Gradle (https://gradle.org/) build file.
+In order to perform a build, all you need is
+a JDK -- version 1.8 or later.
+Clone this Github repository `git clone https://github.com/ome/omero-model.git`.
```
Comment (mtbc) [inline README.md]: GitHub

File: README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+Like any Java project, include the jar into your classpath.
+
+## Build from source
+
+The compilation, testing, launch, and delivery of the application are
+automated by means of a Gradle (https://gradle.org/) build file.
+In order to perform a build, all you need is
+a JDK -- version 1.8 or later.
+Clone this GitHub repository `git clone https://github.com/ome/omero-model.git`.
+Go to the directory and  enter:
```
Comment (mtbc) [inline README.md]: double space

**Comment by jburel** [comment]
Merging so we can fix the doc build. Entries in the changelog will be combined when we release 5.5.0

**Pull request merged**